api-specificatie/drc/openapi.yaml

```diff
@@ -1,57 +1,194 @@
   description: Een API om een documentregistratiecomponent te benaderen
-  version: 0.0.1
   contact:
-    url:  https://github.com/VNG-Realisatie/gemma-zaken
+    url: 'https://github.com/VNG-Realisatie/gemma-zaken'
+    email: support@maykinmedia.nl
+  license:
+    name: EUPL 1.2
+  version: '1'
+security:
+  - basic: []
 paths:
-  '/api/v{version}/informatieobjecten':
+  /enkelvoudiginformatieobjecten:
+    post:
+      operationId: enkelvoudiginformatieobject_create
+      description: Registreer een EnkelvoudigInformatieObject.
+      responses:
+        '201':
+          description: ''
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/EnkelvoudigInformatieObject'
+      tags:
+        - enkelvoudiginformatieobjecten
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/EnkelvoudigInformatieObject'
+        required: true
+    parameters: []
+  '/enkelvoudiginformatieobjecten/{id}':
     get:
-      operationId: informatieobject_list
-      description: Ophalen van informatieobjecten
+      operationId: enkelvoudiginformatieobject_read
+      description: Geef de details van een EnkelvoudigInformatieObject.
       responses:
-        200:
+        '200':
           description: ''
           content:
             application/json:
               schema:
-                type: array
-                items:
-                  $ref: '#/components/schemas/InformatieObject'
+                $ref: '#/components/schemas/EnkelvoudigInformatieObject'
+      tags:
+        - enkelvoudiginformatieobjecten
     parameters:
-      - name: version
+      - name: id
         in: path
+        description: A unique integer value identifying this informatieobject.
         required: true
         schema:
-          type: string
+          type: integer
+  /zaakinformatieobjecten:
+    post:
+      operationId: zaakinformatieobject_create
+      description: Registreer een INFORMATIEOBJECT bij een ZAAK.
+      responses:
+        '201':
+          description: ''
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ZaakInformatieObject'
+      tags:
+        - zaakinformatieobjecten
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/ZaakInformatieObject'
+        required: true
+    parameters: []
+  '/zaakinformatieobjecten/{id}':
+    get:
+      operationId: zaakinformatieobject_read
+      description: Geef de details van een ZAAKINFORMATIEOBJECT relatie.
+      responses:
+        '200':
+          description: ''
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ZaakInformatieObject'
+      tags:
+        - zaakinformatieobjecten
+    parameters:
+      - name: id
```

Comment:
Lastig, in het RGBZ2 heeft een zaakinformatieobject geen identificatie. Het is een relatieklasse die bestaat bij de gratie van een relatie tussen informatieobject en een zaak. 
De ID is deze API is dus puur een technisch ID, geen gegeven uit het RGBZ2. Zou dit iets in een UGM moeten worden? @zomwoof